lib/src/flutter_search_bar_base.dart

```diff
@@ -1,13 +1,12 @@
 // Copyright (c) 2017, Spencer. All rights reserved. Use of this source code
 // is governed by a BSD-style license that can be found in the LICENSE file.
 
-import 'dart:async';
```

Comment:
It compiles and works. "import 'dart:async';" is marked as unused import in Intellij Idea.

Reply:
According to https://github.com/dart-lang/sdk/issues/26162
the async keywords - async, await, yield are available without "import 'dart:async';" at least since Apr 1, 2016

Reply:
Huh, alright! Guess when I wrote this my machine had an old version of dart. I am seeing the same behavior in IntelliJ.